sourcefinder/stats.py

```diff
@@ -7,58 +7,26 @@
 from numpy.ma import MaskedArray
 from scipy.special import erf
 from scipy.special import erfcinv
+from scipy.optimize import fsolve
 
 from .utils import calculate_correlation_lengths
 
 
 # CODE & NUMBER HANDLING ROUTINES
 #
-def var_helper(N):
-    """Correct for the fact the rms noise is computed from a clipped
-    distribution.
-
-    That noise will always be lower than the noise from the complete
-    distribution.  The correction factor is a function of the computed
-    rms noise only.
-    """
-    term1 = numpy.sqrt(2. * numpy.pi) * erf(N / numpy.sqrt(2.))
-    term2 = 2. * N * numpy.exp(-N ** 2 / 2.)
-    return term1 / (term1 - term2)
-
+def find_true_std(sigma, clip_limit, clipped_std):
+    help1 = clip_limit/(sigma*numpy.sqrt(2))
+    help2 = numpy.sqrt(2*numpy.pi)*erf(help1)
+    return sigma**2*(help2-2*numpy.sqrt(2)*help1*numpy.exp(-help1**2))-clipped_std**2*help2
```

Comment:
this is not pep8 compatible formatting

Reply:
My IDE (PyCharm) indeed indicated that two blank lines are required after each function, according to PEP8.
Is that what you mean?